cmake/Hunter/config.cmake

```
@@ -33,3 +33,9 @@ hunter_config(
     SHA1 152b849610d91a9dfa1401293f43230c2e0c33f8
     CMAKE_ARGS BUILD_GMOCK=OFF gtest_force_shared_crt=ON
 )
+
+hunter_config(
+    libscrypt
+    VERSION 1.21-p1
```

```suggestion
    VERSION ${HUNTER_libscrypt_VERSION}
```
Not to override the default version number.